static/app/components/codeSnippet.tsx

```diff
@@ -3,6 +3,7 @@
 import Prism from 'prismjs';
 import 'prismjs/components/prism-bash.min';
 import 'prismjs/components/prism-python.min';
```

Comment:
for some reason prism freaks out when you try to add prism-php, will investigate more